tasks/publish-pyxis-repository/publish-pyxis-repository.yaml

```diff
@@ -85,6 +88,11 @@ spec:
         fi
 
         DATA_FILE="$(workspaces.data.path)/$(params.dataPath)"
+        RESULTS_DIR="$(workspaces.data.path)/$(params.resultsDirPath)"
```

Comment:
This and the following line can be combined into one

Reply:
Done!